src/main/webapp/script/postcardgen.js

```diff
@@ -52,29 +52,126 @@ class Postcard {
     return postcard;
   }
 
+  addImageEmailStyle() {
+    let image = document.createElement('img');
+    image.setAttribute('src', this._imageUrl);
+    image.style.height = '200px';
+    image.style.width = '250px';
+
+    return image;
+  }
+
+  addMessageEmailStyle() {
+    let message = document.createElement('div');
+    message.style.display = 'inline-block';
+    message.style.fontFamily = "'Comic Sans MS', cursive, sans-serif";
+    message.style.fontSize = '30px';
+    message.style.maxWidth = '300px';
+    message.style.width = '250px';
+    message.appendChild(document.createTextNode(this._message));
+
+    return message;
+  }
+
+  addTitleEmailStyle() {
```

Comment:
is it possible to put the style (which isn't dynamic) into the stylesheet, and just add a class attribute here?

Reply:
I did a search right now and this is possible. At the time of the implementation I didn't know is was possible.